be/src/io/fs/s3_file_reader.cpp

```diff
@@ -43,6 +44,9 @@ bvar::Adder<uint64_t> s3_file_reader_read_counter("s3_file_reader", "read_at");
 bvar::Adder<uint64_t> s3_file_reader_total("s3_file_reader", "total_num");
 bvar::Adder<uint64_t> s3_bytes_read_total("s3_file_reader", "bytes_read");
 bvar::Adder<uint64_t> s3_file_being_read("s3_file_reader", "file_being_read");
+bvar::LatencyRecorder s3_bytes_per_read("s3_file_reader", "bytes_per_read"); // also QPS
```

Comment:
It seems calculating the rate of s3_bytes_read_total (throughput) and s3_file_reader_read_counter (qps)
will do the work.
Adding redundant metrics is a burden for the scraping procedure (Prometheus).

Reply:
bvar has fine granularity, so it is easier to identify throughput issue (whether caused by long wait or low instantaneous speed). So I insist, but we can delete them when the optimization work is done.